CHANGES.md

```diff
@@ -24,6 +24,13 @@ this file to understand what changed.
 - bug fixes
 - other
 
+## v0.2.2
+
+Update to `raiwidgets` package.
+
+- Bug fix:
+  - Fix cagegorical feature in ice plot
```

Comment:
nit: categorical features

Reply:
is it possible to merge this indexing bug fix before release:
https://github.com/microsoft/responsible-ai-widgets/pull/352